app/mapping/registration/TrustRegistrationModels.scala

```diff
@@ -155,9 +155,12 @@ object TrusteeOrgType {
 }
 
 case class TrusteeIndividualType(name: FullName,
-                       dateOfBirth: Option[LocalDate],
-                       phoneNumber: Option[String],
-                       identification: Option[IdentificationType])
+                                 dateOfBirth: Option[LocalDate],
+                                 phoneNumber: Option[String],
+                                 identification: Option[IdentificationType],
+                                 nationality: Option[String],
+                                 countryOfResidence: Option[String],
+                                 legallyIncapableYesNo: Option[Boolean])
```

Comment:
Legally incapable field need to match the backend for validation to succeed
https://github.com/hmrc/trusts/blob/622ac9a090709d91c45df9142bf80a16908dcdf0/conf/resources/schemas/5MLD/trusts-api-registration-schema-1.3.0.json#L583-L612

Reply:
Yep, this is the bit I hadn't worked on yet. Done now